src/components/SingleArticlePage.js

```diff
@@ -17,31 +20,47 @@ const SingleArticlePage = () => {
     <>
       {article && (
         <div className="articles_single-page-article-container">
+          <h1 id="single-page_title">{article.article.title}</h1>
```

Comment:
Please take a look at where article is coming from and destructure it one more level so we don't have article.article! You should be able to do this in the useEffect, or in your api file.